spark_expectations/sinks/utils/writer.py

```diff
@@ -439,6 +421,7 @@ def _prep_detailed_stats(
                 "table_name",
                 "rule_type",
                 "rule",
+                "column_name",
```

Comment:
are we making this column a mandatory and is this going to be a breaking change for the existing pipelines? I think it would be better to makes this optional for the next release and then make it mandatory later as folks adopt it

Reply:
yes, for now I'm reverting back the additional column changes and going forward with the duplicate check fixes.